examples/basic/mod.ts

```diff
@@ -0,0 +1,15 @@
+import { DOMParser, Element } from "https://deno.land/x/deno_dom@v0.1.2-alpha4/deno-dom-native.ts";
```

Comment:
The examples are a nice idea, just that I'd import from `https://deno.land/x/deno_dom/deno-dom-wasm.ts` here.

Reply:
This should also be
```typescript
import { DOMParser } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
import type { Element } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
```